lib/is-windows.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function () {
+  return process.platform === 'win32' ||
+    process.env.OSTYPE === 'cygwin' ||
```

Comment:
@rmg I believe I cargo culted this from somwhere, it might have been that I pulled the logic originally sitting in `index.js` to `is-windows.js` -- my guess would be there's a good reason for the `cygwin` check, it could also be that someone accidentally created a cult and I've been indoctrinated into it.